packages/pnpm-sync-lib/src/pnpmSyncPrepare.ts

```diff
@@ -29,6 +30,11 @@ export interface IPnpmSyncPrepareOptions {
    */
   dotPnpmFolder: string;
 
+  /**
+   * An identifier that can be used to recognize the `pnpm-lock.yaml`
+   */
+  identifier?: string;
```

Comment:
(The word "identifier" is slightly vague as to its purpose. Some other ideas: `consumer`, `lockfileId`, `lockfileName`, `pruningKey`, `installation`, etc.  If you don't feel strongly about it, I'm okay with keeping "identifier.")

Reply:
`lockfileId ` is a good name, and make more sense since the  purpose of `identifier` is to recognize the `pnpm-lock.yaml`